Makefile

```diff
@@ -55,9 +55,6 @@ testenv:
 	go run cmd/temporal/main.go -config ./testenv/config.json --db.no_ssl migrate
 	make api-user
 	make api-admin
-	# create minio bucket
-	make bucket
-	make bucket BUCKET=test
```

Comment:
i didnt see any mention of buckets in the PR desc, are we dropping minio usage completely?